src/Avalonia.Headless/HeadlessWindowImpl.cs

```diff
@@ -199,7 +199,9 @@ public void Dispose()
 
         public ILockedFramebuffer Lock()
         {
+#pragma warning disable CS0618
             var bmp = new WriteableBitmap(PixelSize.FromSize(ClientSize, RenderScaling), new Vector(96, 96) * RenderScaling);
+#pragma warning restore CS0618
```

Comment:
```suggestion
            var bmp = new WriteableBitmap(PixelSize.FromSize(ClientSize, RenderScaling), new Vector(96, 96) * RenderScaling, PixelFormat.Rgba8888, AlphaFormat.Premul);
```
which are defaults for Headless https://github.com/AvaloniaUI/Avalonia/blob/f51efd3318ae6413fa83c78248b8afd860e5ab7b/src/Avalonia.Headless/HeadlessPlatformRenderInterface.cs#L26-L28